job-dsls/jobs/downstream_pr_jobs.groovy

```diff
@@ -171,6 +171,7 @@ for (repoConfig in REPO_CONFIGS) {
                         mavenHome("/opt/tools/apache-maven-${Constants.MAVEN_VERSION}")
                         delegate.mavenOpts("-Xmx3g")
                         mavenArgs(get("downstreamMvnGoals") + " " + get("downstreamMvnProps").collect { k, v -> "-D$k=$v" }.join(" "))
+                        providedSettings("settings-local-maven-repo-nexus")
```

Comment:
this should be add to  project / 'builders' << 'hudson.tasks.Maven' { - this specifies Invoke Top Level Maven targets - this is the only place where you can (in advanced) specify a settings.xml.